src/Command/ProcessCommand.php

```diff
@@ -85,17 +79,19 @@ protected function configure(): void
     {
         $this->addOption('sync-configuration', null, InputOption::VALUE_NONE, 'Sync plugin configuration with database');
 
+        $syncConfigCommand = SynchronizeVariantConfigurationCommand::getDefaultName();
+
         $this->setHelp(
-            <<<'EOF'
+            <<<'EOT'
```

Comment:
What's the difference between EOF and EOT? I think I just copied this originally from Symfony so I don't know anything about it :)

Reply:
I think, actually, it doesn't really matter what you choose - you can choose whatever you want, as long as whatever comes after `<<<` is the same as what you end it with. I just chose EOT because I saw a snippet with string interpolation using those delimiters, and I thought that it matters.

It seems we have a [nowdoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc) here, when we in fact want a [heredoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc) since it allows string interpolation for the command name.

Reply:
Never researched that. Good to know :D